app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MediaRanker</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    <header>
+    <h1><%= link_to 'Media Ranker', '/' %><small>Ranking the Best of Everything</small></h1>
+    <%= link_to 'Log In', login_path %>
+    </header>
+    <%= flash[:message] %>
```

Comment:
I would like to see a lot more functionality in this layout:
- A navbar
- Better support for letting the user know about their login status
- More structure around displaying flash messages